Assets/Scripts/Game/EnemySenses.cs

```diff
@@ -715,9 +715,9 @@ void GetTargets()
                         if (targetEntity != null && targetEntity.MobileEnemy.Team == enemyEntity.MobileEnemy.Team)
                             continue;
                     }
-                    else // TODO: Support player-summoned allies, even without infighting option
+                    else
                     {
-                        if (targetBehaviour != Player)
+                        if (targetBehaviour != Player && !mobile.Summary.Enemy.AlliedToPlayer)
                             continue;
```

Comment:
There will be merge conflicts with https://github.com/Interkarma/daggerfall-unity/pull/1231 because I modified this same area of code.